### DISCUSSION TITLE
Constraints?

### DISCUSSION THREAD

**Discussion by tpoisot** [MilesCranmer/PySR#304] [Q&A] [answered]
Is there a way to set some constraints on the results? I have a specific problem for which I know the solution must be positive, is there a way to indicate this?

Other than that - great job on the package. Super fast, super intuitive to use. A game changer.

**Comment by MilesCranmer** [comment]
Hi @tpoisot, sorry for my very late response! Just seeing this. Thanks for your kind words.
I think this functionality would be nice to implement. Do you want to join the thread here https://github.com/MilesCranmer/PySR/discussions/69? Someone asked a related question and I described how it could be implemented.

Cheers!
Miles

**Comment by MilesCranmer** [comment, marked as answer]
Update: added `full_objective` (PySR) and `loss_function` (SymbolicRegression.jl) for this purpose.
